src/dash-playlist-loader.js

```diff
@@ -1,5 +1,5 @@
 import { EventTarget, mergeOptions } from 'video.js';
-import mpdParser from 'mpd-parser';
+import { parse as mpdParser, parseUTCTiming } from 'mpd-parser';
```

Comment:
Minor, but in the interest of a verb for the function, maybe `parse as parseMpd`?